templates/aws-stack.yml

```
@@ -587,6 +587,21 @@ Parameters:
       - "dualstack"
     Default: "ipv4"
 
+  DockerIPv4AddressPool1:
+    Type: String
+    Description: Primary IPv4 CIDR block for Docker default address pools. Only applies to Linux instances, not Windows.
```

Nit, but I think we could make description a bit more clear, 
```suggestion
    Description: Primary IPv4 CIDR block for Docker default address pools. Must not conflict with host network or VPC CIDR. Only applies to Linux instances, not Windows.
```